### PR TITLE
Add awscli image

### PR DESCRIPTION
**Pull request by clook**

### CODE REVIEW

File: awscli/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM alpine:edge
```
Comment (david-berichon) [inline awscli/Dockerfile]: sure about the edge case ?

**Comment by tauffredou** [comment]
To be discussed : 
- one image per tool
- one tools image

File: awscli/Dockerfile (hunk 1 above)
Comment (clook) [reply awscli/Dockerfile]: You're ritght, just switched to latest

**Comment by clook** [comment]
> To be discussed :
> 
>     * one image per tool
> 
>     * one tools image
* both

Sure, we'll have to speak about the many cases and why :)

**Pull request merged**